Software/Python/setup.py

```diff
@@ -19,7 +19,7 @@
 
 setup(
     name = "gopigo3",
-    version = "1.0.2",
+    version = "1.1.0",
```

Comment:
I've added this increment here because there are other changes in other PRs too. Think of mutex, the motionsensor, documentation, etc.